tests/integration/launcher.ts

```diff
@@ -49,5 +49,8 @@ describe('Elyra launcher is in use', () => {
     cy.get(
       '.jp-LauncherCard[data-category="Elyra"][title="Documentation"]:visible'
     );
+    cy.get(
+      '.jp-LauncherCard[data-category="Elyra"][title="What\'s new in latest"]:visible'
```

Comment:
I believe this will break when CI tests are run on a branch other than 'main'. For example, assume we release 3.11.0 and create a new Git tag. If we need to create a `3.11.1` patch release in the future (one that is not based on the 'main' branch), the title will be `What's new in v3.11.0` because the release script modified the source code.

Reply:
Updated test to check for substring instead